plugins/builders/CadsBuilder.rb

```diff
@@ -50,7 +56,6 @@ def load_cads
       end
       cads_loader.setup
       cads_loader.eager_load
-      # Rails.application.config = CitizensAdviceComponents::Engine.config
```

Comment:
I think this part was only required for Primer because that has some config items https://github.com/primer/view_components/blob/main/lib/primer/view_components/engine.rb#L17-L19